packages/opal-common/opal_common/git/commit_viewer.py

```diff
@@ -210,6 +210,29 @@ def directories(self, filter: Optional[DirectoryFilter] = None) -> Generator[Ver
                 if filter(directory):
                     yield directory
 
+    def get_node(self, path: Path, filterable_gen: Optional[Callable] = None) -> Optional[VersionedNode]:
+        """
+        Returns the node in the given path, None if it doesn't exist
+        """
+        filterable_gen = filterable_gen or self.nodes
+        path = Path(path) # Works even if path is already Path
```

Comment:
Are you expecting `path` to maybe _not_ be a `Path`? Maybe some Mypy typechecking would help here...